pkg/didcomm/protocol/introduce/service.go

```diff
@@ -478,7 +398,7 @@ func stateFromName(name string) state {
 // canTriggerActionEvents checks if the incoming message can trigger an action event
 func canTriggerActionEvents(msg service.DIDCommMsg) bool {
 	switch msg.Type() {
-	case ProposalMsgType, ResponseMsgType, RequestMsgType:
+	case ProposalMsgType, RequestMsgType:
```

Comment:
Is this change part of refactor ? This looks like logic change.

Reply:
The logic was changed. It looks like ResponseMsgType is redundant and no needed. It only makes service and usage complicated. I don't see any benefits of this action. @llorllale Could you please confirm that this action is redundant?

Reply:
@soluchok I think so. However, is this the only state for which the framework is skipping action events (including did-exchange)? If it is, it will stick out like a sore thumb

Reply:
Not sure about did-exchange, but there is no reason to execute the Continue function. Currently, the Continue function is executed when:
* need to accept the proposal (introducee may provide invitation)
* need to accept the request (introducer should provide invitation or send a proposal to another introducee)

It doesn't make sense to reject or accept the `response`.

Reply:
I can't really argue one way or the other at this point in time